fdb-record-layer-core/src/main/java/com/apple/foundationdb/record/provider/foundationdb/OnlineIndexer.java

```diff
@@ -525,6 +535,22 @@ public void close() {
         }
     }
 
+    private void maybeLogBuildProgress(Subspace subspace, Tuple startTuple, Tuple endTuple, Tuple realEnd) {
+        if (LOGGER.isInfoEnabled()
+            && (progressLogIntervalMillis > 0
+                && System.currentTimeMillis() - timeOfLastProgressLogMillis > progressLogIntervalMillis)
+            || progressLogIntervalMillis == 0) {
+                LOGGER.info(KeyValueLogMessage.of("Built Range",
+                        "indexName", index.getName(),
+                        "indexVersion", index.getLastModifiedVersion(),
+                        "subspace", subspace,
```

Comment:
When @dsimmen's change goes in for https://github.com/FoundationDB/fdb-record-layer/pull/415 we'll be able to log the subspace provider, which would be much nicer, but until then I guess this is the best we can do.